ci/after_success.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+set -xe
+
+test "$AFTER_SCRIPT" = "1" || exit 0
+
+# Submit code coverage report
+beaver run ci/codecov.sh
```

Comment:
This file is not commited. perhaps we can just drop this for now (remove entire file and also remove after_success from `.travis.yml`